.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,34 @@
+name: reviewdog-suggester
+
+on:
+  pull_request: {}
+
+jobs:
+  black:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    - name: Install
+      run: |
+        python -m pip install -U pip
+        pip install --progress-bar off -U .[checking]
+
+    - name: Black
+      run: black .
+
+    - name: Blackdoc
+      run: blackdoc .
+
+    - name: Isort
+      run: isort .
+
+    - name: Reviewdog
+      uses: reviewdog/action-suggester@v1
+      with:
+        tool_name: black
```

Comment:
Should we change this value as well?

Reply:
👍 

Before: https://github.com/himkt/optuna-black-suggester/pull/5
After: https://github.com/himkt/optuna-black-suggester/pull/6

Reply:
lovely.